Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -143,12 +144,8 @@ public virtual bool CanLaunchWebBrowser
                     return false;
                 }
 
-                if (EnvironmentInfo.OperatingSystem == MediaBrowser.Model.System.OperatingSystem.Windows)
-                {
-                    return true;
-                }
-
-                if (EnvironmentInfo.OperatingSystem == MediaBrowser.Model.System.OperatingSystem.OSX)
+                if (OperatingSystem.Id == OperatingSystemId.Windows
+                    || OperatingSystem.Id == OperatingSystemId.Darwin)
```

Comment:
Shouldn't the or operator be on the previous line?

Reply:
Afaik both are used in the code, but newline is dominant and the analyzers prefer it.